internal/server/handler_v1.go

```diff
@@ -550,3 +550,29 @@ func (s *Server) SearchStatVar(
 
 	return merged, nil
 }
+
+// FilterStatVarsByPlace implements API for Mixer.FilterStatVarsByPlace.
+func (s *Server) FilterStatVarsByPlace(
+	ctx context.Context, in *pb.FilterStatVarsByPlaceRequest,
+) (*pb.FilterStatVarsByPlaceResponse, error) {
+	localResp, err := search.FilterStatVarsByPlace(ctx, in, s.store, s.cachedata.Load())
```

Comment:
Can the local/remote paths be done concurrently? Something like https://github.com/datacommonsorg/mixer/blob/2214b4ee93ab9ed6a947da73d231c0b3f19cfe08/internal/server/handler_v1.go#L196